<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>800</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>290</width>
      <height>78</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="topBar">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="serverInfos">
       <item>
        <layout class="QHBoxLayout" name="cmdServer">
         <item>
          <widget class="QLabel" name="cmdLabel">
           <property name="text">
            <string>CMDServer </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cmdAddr"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="videoServer">
         <item>
          <widget class="QLabel" name="videoLabel">
           <property name="text">
            <string>VideoServer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="videoAddr"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="connectBtn">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
